mod-pubsub-server/src/main/java/org/folio/services/audit/AuditService.java

```diff
@@ -19,7 +19,7 @@ static AuditService create() {
   }
 
   static AuditService createProxy(Vertx vertx) {
-    return new AuditServiceVertxEBProxy(vertx, AUDIT_SERVICE_ADDRESS);
+    return new org.folio.services.audit.AuditServiceVertxEBProxy(vertx, AUDIT_SERVICE_ADDRESS);
```

Comment:
Why do we need a fully qualified class name?

Reply:
It was a problem to built it without these changes. If it is not needed we can revert changes

Reply:
reverted back